types/lodash/index.d.ts

```diff
@@ -12269,7 +12269,7 @@ declare namespace _ {
          * @param value The value to inspect.
          * @return Returns true if value is empty, else false.
          */
-        isEmpty(value?: any): boolean;
+        isEmpty(value?: any): value is null | undefined;
```

Comment:
Yes indeed, but that would he caught at runtime through the JS logic, as it cannot be caught during compilation. Isn't the return type atill a boolean anyways?